documentation/source/newsfragments/README.rst

```diff
@@ -0,0 +1,50 @@
+:orphan:
+
+Release Notes
+=============
+
+We are using `towncrier <https://pypi.org/project/towncrier/>`_ to handle
+our release notes, and this directory contains the input for it -
+"news fragments" which are short files that contain a small
+**ReST**-formatted text that will be added to the next version's
+Release Notes page.
+
+Each file should be named like ``<PULL REQUEST>.<TYPE>.rst``, where
+``<PULL REQUEST>`` is a pull request number, and ``<TYPE>`` is one of:
```

Comment:
In many cases, the issue is the start and end of a discussion, and the PR only implements it. Of course, that's not always true, and not every PR has an associated issue.

In case where the issue is the main center of a discussion (as it should be), then the issue number is the one to use here. Can you reword that somehow to allow both issue and PR numbers?

Reply:
What matters most is that the number is unique, but agreed that sometimes the issue is more valuable.

It's usually trivial to get from the PR to the issue though through github anyway, so I'd be inclined to leave this advice as is.